src/mz-debug/src/utils.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License in the LICENSE file at the
+// root of this repository, or online at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs::{create_dir_all, remove_dir_all, File};
+use std::io::{copy, BufWriter};
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use chrono::{DateTime, Utc};
+use zip::write::SimpleFileOptions;
+use zip::ZipWriter;
+
+static DEBUG_FOLDER: &str = "mz-debug";
```

Comment:
You are naming the debug directory the same thing as your binary. If they are in the same directory, they will conflict.

Reply:
Good call!